### PR TITLE
Create Prims_Algorithm.cpp

### PR DESCRIPTION
**Pull request by MaYaNkKashyap681**
Prims Algorithm in C++.

### CODE REVIEW

**Review by Kumar-laxmi — COMMENTED** [review]
Kindly update the repository **README.md**

**Comment by MaYaNkKashyap681** [comment]
Ok I will update it, is my code is ok or you need some changes.

**Comment by MaYaNkKashyap681** [comment]
Why i am getting conflict can you please help me with this.

File: README.md [hunk 1]
```diff
@@ -54,6 +54,7 @@
 | 17.  | Binary Search Tree | [C](https://github.com/Kumar-laxmi/Algorithms/tree/main/C/Trees) | Trees |
 | 18.  | Red-Black Tree | [C++](https://github.com/Kumar-laxmi/Algorithms/blob/main/C%2B%2B/Trees/red_black.cpp), [Python](https://github.com/Kumar-laxmi/Algorithms/blob/main/Python/Trees/red_black.py) | Trees |
 | 19.  | Decision Tree | [Python](https://github.com/Kumar-laxmi/Algorithms/blob/main/Python/Machine%20Learning/Decision%20Tree/decision_tree.py) | Machine Learning |
+| 20.  | Prim's Algorithm | [C++](https://github.com/Kumar-laxmi/Algorithms/blob/main/C%2B%2B/Graphs/Prims_Algorithm.cpp) | Graphs |
```
Comment (Kumar-laxmi) [inline README.md:57]: Please put Prim's Algorithm between Floyd Warshall(Index 3) and Singly Linked List(Index 4)

**Review by Kumar-laxmi — CHANGES_REQUESTED** [review]

**Comment by MaYaNkKashyap681** [comment]
Requested Change Resolved.

**Pull request merged**